### PR TITLE
Project Version Update, main branch (2024.08.15.)

### PR DESCRIPTION
**Pull request by krasznaa**
Update project to version `0.15.0`, in preparation for the next tag.

@paulgessinger, @beomki-yeo, let's keep attention to this. I don't like it when a tagged release advertises itself as a different version after being built. :thinking:

### CODE REVIEW

**Review by beomki-yeo — APPROVED** [review]
I agreee with you but this bothers me a lot at the same time. I wish the version number is updated automatically...

**Comment by krasznaa** [comment]
This setup (where we pre-setup the version number) is the least work-prone procedure in my mind. :thinking:

Indeed, projects often have a "release procedure" that involves setting the project version as part of the procedure. Some of that could even be automated I guess. But as far as I know, none of that comes for free. Even in the best of cases we'd need to write some helper scripts that could be used during the release making procedure.

Long story short, I think this is the least painful of all the options for this project. I'm open to suggestions though...
